server/service/personalization.jsx

```diff
@@ -135,7 +134,7 @@ function contentToJSX(content : string) : ComponentFunctionType<PersonalizationC
 }
 
 export async function resolvePersonalization(req : ExpressRequest, gqlBatch : GraphQLBatchCall, personalizationOptions : PersonalizationOptions) : Promise<Personalization> {
-    let { logger, clientID, merchantID, locale, buyerCountry, buttonSessionID, currency, intent, commit,
+    let { logger, clientID, locale,  buyerCountry, buttonSessionID, currency, intent, commit,
```

Comment:
nit: there's an extra space where `merchantID` was removed.
```suggestion
    let { logger, clientID, locale, buyerCountry, buttonSessionID, currency, intent, commit,
```